Code/Framework/AzCore/Tests/RTTI/TypeSafeIntegralTests.cpp

```diff
@@ -24,7 +24,7 @@ namespace UnitTest
 
     template <typename IntegralType>
     class TypeSafeIntegralTests
-        : public UnitTest::ScopedAllocatorSetupFixture
+        : public UnitTest::LeakDetectionFixture
     {
         ;
```

Comment:
Odd semicolon in the body of a class.  Remove it.

Reply:
Fixed.